crates/symbolicator/src/services/cacher.rs

```diff
@@ -372,17 +406,148 @@ impl<T: CacheItemRequest> Cacher<T> {
         let key = request.get_cache_key();
 
         // cache_path is None when caching is disabled.
-        let cache_path = self.get_cache_path(&key);
+        let cache_path = self.get_cache_path(&key, T::VERSIONS.current);
         if let Some(ref path) = cache_path {
             if let Some(item) = self.lookup_cache(&request, &key, path)? {
                 return Ok(Arc::new(item));
             }
+
+            // try fallback cache paths next
+            for fallback_path in T::VERSIONS
+                .fallbacks
+                .iter()
+                .flat_map(|version| self.get_cache_path(&key, *version))
+            {
+                if let Ok(Some(item)) = self.lookup_cache(&request, &key, &fallback_path) {
+                    // we have found an outdated cache that we will use right away,
+                    // and we will kick off a recomputation for the `current` cache version
+                    // in a deduplicated background task, which we will not await
```

Comment:
Is it okay to defer this to a followup PR. There is a task to get a metric for recomputations in as well, and I had some questions about that as well. Also NOTE that this PR on its own does nothing, because we don’t use fallback versions anywhere yet (except for tests)

Reply:
Hum, I don't see the benefit to defer this.  Can only see us forgetting it.  Adding a counter should be that much work?